src/hera.h

```diff
@@ -33,6 +33,16 @@ struct evm_instance;
 
 struct evm_instance* hera_create(void);
 
+typedef enum wasm_vm {
+#if WABT_SUPPORTED
```

Comment:
This shouldn't have ifdefs.

Reply:
this is for CMake build option. Why not?

Reply:
What is the reason ifdef enums?

Reply:
hera_instance contains one such enum, and preprocessor also controls which options are available with evm_set_option.

Reply:
I mean the ifdef only needs to be present in `evm_set_option`, there wouldn't be any other way to select them?

Reply:
I don't think it makes a difference at all, but why define constants for VMs that are not built in?